README.md

```diff
@@ -41,7 +42,10 @@ FAQ
 How do I refresh the task report?
 ---------------------------------
 
-Press any key except `y`.  (If you press `y`, the last display will remain,
+If the optional `refresh_every` key is set to a value greater than 0, the task
+report is automaticaly refreshed every specified seconds.
```

Comment:
"every specified seconds" isn't grammatical, is it?  One alternative would be "set to a value `N` greater than 0" and "every `N` seconds" (but feel free to rephrase if needed).